vortex-error/src/lib.rs

```diff
@@ -221,12 +229,16 @@ pub trait VortexUnwrap {
     fn vortex_unwrap(self) -> Self::Output;
 }
 
-impl<T> VortexUnwrap for VortexResult<T> {
+impl<T, E> VortexUnwrap for Result<T, E>
+where
+    E: Into<VortexError>,
+{
```

Comment:
This makes it way easier to do `vortex_unwrap` / `vortex_expect` without needing to do a `map_err` first